docs/_docs/integration-checks-reference.md

```diff
@@ -7,7 +7,13 @@ redirect_from: /docs/
 
 # Integration Checks Reference
 
-We run your integration through a set of automated checks to ensure it's working property and giving our users the best possible experience. To help better address a check in communication, each check is given a unique ID, consisting of a capital letter and three digits, such as `D001`.
+We run your integration through a set of automated checks to ensure it's working properly and giving our users (and yours) the best possible experience. 
+
+To [publish your integration](https://platform.zapier.com/partners/lifecycle-planning#3-publish-your-integration-to-the-app-directory), all Errors and Publishing Tasks should be validated. You can address any Warnings after [submitting your integration for review](https://platform.zapier.com/partners/lifecycle-planning#4-submit-your-integration-for-review-by-the-zapier-team).
```

Comment:
nit: when writing urls, you should exclude the url of the site itself. This way, the links work locally and if we every change the domain this site runs on, we won't have to tweak them then.

> To [publish your integration](/partners/lifecycle-planning#3-publish-your-integration-to-the-app-directory), all Errors ...

Reply:
Thanks for the suggestion, @xavdid. I took note of it. I'll leave the implementation for next time, given that the rest of the urls in the doc have urls this way. Next time I update this doc, I'll re-consider fixing it for all of them.